python-cffi/netplan/_build_cffi.py

```diff
@@ -68,6 +68,7 @@
     // Parser
     NetplanParser* netplan_parser_new();
     void netplan_parser_clear(NetplanParser **npp);
+    void netplan_parser_set_flags(NetplanParser *npp, int flags);
```

Comment:
suggestion: we probably want to expose a `netplan_parser_get_flags(npp)` already.

This would give us the liberty to modify the flags as side effects of other operations, like when we're pre-loading data, etc... as the caller wouldn't need to keep an external copy of the flags and could just do `netplan_parser_set_flags(npp, netplan_parser_get_flags(npp) | NETPLAN_PARSER_IGNORE_ERRORS);`